docs/CONTRIBUTING.md

```diff
@@ -66,6 +83,22 @@ After you are able to successfully build JsDbg, please follow the steps below to
 
 8. The JsDbg server will launch and prompt you to select a browser. If the debugger is attached to one browser, it is recommended that you choose a different browser for JsDbg.
 
+#### GDB
+
+1. Make sure you added JsDbg to your `.gdbinit` as described in the Linux build instructions above
+
+2. Type `jsdbg` at the gdb prompt to run it. It will automatically open a browser tab for JsDbg.
+
+3. Extensions should be automatically reloaded when you edit the source file; no need to restart the server. You can also reload the Python script without restarting if needed:
```

Comment:
At the end of the second sentence (about reloading without restart), I think we need to add something along the lines of "by running the code below". Also, would be good to clarify where to run the code (gdb prompt? shell?).

Reply:
Done